spec/feature/admin_page_spec.rb

```diff
@@ -28,7 +28,7 @@
     visit "/feedback/new"
     expect(page).to have_content("Tell us about your experience with Caseflow")
     expect(page).to have_css("#feedback_feedback")
-    page.should have_link("Cancel")
```

Comment:
This was an old syntax and it was giving deprecation warnings. Same applies to other spec tests below.